server/sonar-web/src/main/js/components/common/SelectList.js

```diff
@@ -131,22 +133,27 @@ export default class SelectList extends React.PureComponent {
     return { active: props.items[idx - 1] };
   };
 
+  renderChild = (child /*: Object */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)